doc/docs.md

```diff
@@ -5314,7 +5314,7 @@ struct C.Foo {
 }
 
 // Used to add a custom calling convention to a function, available calling convention: stdcall, fastcall and cdecl.
-// This list aslo apply for type aliases (see below).
+// This list also apply for type aliases (see below).
```

Comment:
If you don't mind fixing this while you're there... it didn't trigger because the word as-is is spelled correctly, it's just wrong grammatically...
```suggestion
// This list also applies for type aliases (see below).
```